### PR TITLE
feat(dedicated): modify error description

### PR DESCRIPTION
**Pull request by sachinrameshn**
| Question         | Answer
| ---------------- | ---
| Branch?          | `MANAGER-12381` 
| Bug fix?         | no
| New feature?     | yes
| Breaking change? | yes/no
| Tickets          | MANAGER-13679 
| License          | BSD 3-Clause

- [x] Try to keep pull requests small so they can be easily reviewed.
- [x] Commits are signed-off
- [x] Only FR translations have been updated
- [x] Branch is up-to-date with target branch
- [x] Lint has passed locally
- [x] Standalone app was ran and tested locally
- [x] Ticket reference is mentioned in linked commits (internal only)
- [ ] Breaking change is mentioned in relevant commits

### CODE REVIEW

File: packages/manager/apps/dedicated/client/app/ip/translations/Messages_fr_FR.json [hunk 1]
```diff
@@ -237,6 +237,8 @@
   "ip_byoip_aggregate_block_title": "Agréger le bloc",
   "ip_byoip_aggregate_set_parent_size": "Veuillez spécifier le bloc parent désiré",
   "ip_byoip_aggregate_configuration": "Cette configuration entrainera la création du bloc parent suivant :",
+  "ip_byoip_aggregate_slicing_not_available_error_message": "Aucune configuration de segmentation/agrégation disponible. Pour plus d’informations, veuillez consulter notre guide : <a href=\"{{ url }}\" target=\"_blank\" rel=\"noopener\">ici</a>",
+  "ip_byoip_aggregate_ip_doesnt_exist_error_message": "Une autre action de segmentation ou d’agrégation a été effectuée pour cette adresse IP.",
```
Comment (ghyenne) [inline packages/manager/apps/dedicated/client/app/ip/translations/Messages_fr_FR.json:241]:
```suggestion
  "ip_byoip_aggregate_ip_doesnt_exist_error_message": "Une autre action de segmentation ou d'agrégation a été effectuée pour cette adresse IP.",
```

**Review by qpavy — APPROVED** [review]

File: packages/manager/apps/dedicated/client/app/ip/components/list/byoip/constants.js [hunk 2]
```diff
@@ -1,3 +1,48 @@
...
+  GB:
+    'https://help.ovhcloud.com/csm/en-gb-documentation-network?id=kb_browse_cat&kb_id=74d98129a884a950f07829d7d5c7523d',
+  CA:
+    'https://help.ovhcloud.com/csm/en-ca-documentation-network?id=kb_browse_cat&kb_id=74d98129a884a950f07829d7d5c7523d',
+  US: 'https://us.ovhcloud.com/support/',
+  MA:
+    'https://help.ovhcloud.com/csm/fr-ma-documentation-network?id=kb_browse_cat&kb_id=74d98129a884a950f07829d7d5c7523d',
+  SN:
+    'https://help.ovhcloud.com/csm/fr-sn-documentation-network?id=kb_browse_cat&kb_id=74d98129a884a950f07829d7d5c7523d',
+  TU:
```
Comment (anooparveti) [inline packages/manager/apps/dedicated/client/app/ip/components/list/byoip/constants.js:30]:
```suggestion
  TN:
```

File: packages/manager/apps/dedicated/client/app/ip/components/list/byoip/aggregate/ip-ip-byoip-aggregate.controller.js [hunk 3]
```diff
@@ -12,19 +20,27 @@ export default class IpByoipAggregateController {
...
+    this.showSlicingNotAvailableError = false;
     this.atInternet.trackPage({ name: TRACKING_PREFIX_AGGREGATE });
     this.IpByoipService.getAvailableAggregationConfigurations(this.ip.ipBlock)
       .then(({ data }) => {
         this.aggregationIps = data;
         if (this.aggregationIps.length) {
           [this.selectedAggregationIp] = this.aggregationIps;
+        } else {
+          this.showErrorMessage = true;
+          this.showSlicingNotAvailableError = true;
```
Comment (anooparveti) [inline packages/manager/apps/dedicated/client/app/ip/components/list/byoip/aggregate/ip-ip-byoip-aggregate.controller.js:39]: **nitpick:** Pretty confusing with both the variables. Can we try. and simplify the implementation with single variable.

**Review by anooparveti — CHANGES_REQUESTED** [review]

File: packages/manager/apps/dedicated/client/app/ip/components/list/byoip/aggregate/ip-ip-byoip-aggregate.controller.js (hunk 3 above)
Comment (sachinrameshn) [reply packages/manager/apps/dedicated/client/app/ip/components/list/byoip/aggregate/ip-ip-byoip-aggregate.controller.js]: Remove it now.

**Review by anooparveti — APPROVED** [review]

**Pull request merged**